test-result-summary-client/src/Build/Summary/ResultSummary.jsx

```diff
@@ -102,15 +102,15 @@ export default class ResultSummary extends Component {
                 } else if (suffix === 'temurin') {
                     jdkImpl = 'hs';
                 }
-                if (buildName.includes('openj9')) {
-                    jdkImpl = 'j9';
-                }
```

Comment:
Should we keep L105-L107?

Reply:
Feels like dup of https://github.com/adoptium/aqa-test-tools/blob/74f84e95265b3754868954a3d9cd82340dc1b6a9/test-result-summary-client/src/Build/Summary/ResultSummary.jsx#L100-L101 unless ibm jdk builds have different name conventions.